L1Trigger/L1CaloTrigger/python/Phase1L1TJetCalibrator_cfi.py

```diff
@@ -0,0 +1,783 @@
+import FWCore.ParameterSet.Config as cms
+calibration = cms.VPSet(
+    cms.PSet(
+        etaMax = cms.double(0.435),
+        etaMin = cms.double(0),
+        l1tCalibrationFactors = cms.vdouble(
+            1.30593500928, 1.30593500928, 1.30593500928, 1.30593500928, 1.30593500928, 
```

Comment:
are these values expected to change/evolve? might be better to put them in a text or binary format in a data repo